web/src/app/styles/materialStyles.js

```diff
@@ -120,4 +120,24 @@ export const styles = theme => ({
   mobileGridSpacing: {
     marginBottom: '1em',
   },
+  smallestSubtitle: {
+    fontSize: '1rem',
+    fontFamily: 'Roboto, Helvetica, Arial, sans-serif',
```

Comment:
@mastercactapus @Auchindoun The best way IMO to solve this without hardcoding anything is to use the variants as we were using them before from material UI and updating the components. I took this approach because I thought you wanted me to avoid using the variants.

Reply:
The issue wasn't with the variants directly. IMO there's no good way for a developer to know what variants to use for common use cases. With this, a developer would need to know css properties and values, which would be _more_ confusing.

Variants might be the proper way to interface with the material library, but if we have common cases it would probably make sense to create a re-usable component whos "defaults" apply the proper variant.

Reply:
Based on the files changed, do you see a pattern? Most of the variants are set to subtitle1 or h5, but neither seems to have a general use case for it. Like subtitle1 is used in headings and paragraph text.

Reply:
If nobody has any ideas on how to solve it, we may just need to go with it for now and hope for the best. The only thing I can think of would be to use contexts or something so that it "picks" the right tag based on where it's rendered. But I don't think that would be worth the effort/complexity involved.

I just want to be deliberate as I think it's highly unlikely that any future/refactored code will be consistent in the future without solving it in an intuitive way, so we just need to be cognizant of the maintenance involved to keep things passing.

It may be less of an issue once we have validation in the CI pipeline too. Anyway, I'm ok with the variant attributes if that's the best approach we have, it's at least obvious what is happening even if the "why" requires a fair amount of context.